bin/nv

```diff
@@ -41,7 +46,7 @@ require('yargs')
           })
         }).catch(e => {
           console.error(e)
-          process.exitCode = e.code || 1
+          process.exitCode ||= e.code || 1
```

Comment:
Hm, while I *think* I agree with this change it actually makes me wonder if this should instead handle `process.exit` itself?

Reply:
it could, sure - but this way it will still exit the same, except that it’ll preserve a preexisting nonzero exit code.

Reply:
yeah, that's why I was on board with the change. It just got me thinking about the other thing. If something sets and exit code but doesn't exit I always feel like that is an "action at a distance" which I dont like.

Reply:
i 100% agree but long ago node shifted into "set the exit code" as idiomatic instead of "exit with a code", and i've given up fighting it :-)

Reply:
hm, interesting. I would love to read more about that shift. Anyway, I didn't want to block on this either way. Seems like this is good to merge?

Reply:
Go for it!